src/Common/tests/System/Net/Sockets/TestSettings.cs

```diff
@@ -11,6 +11,11 @@ public static class TestSettings
         public const int FailingTestTimeout = 100;
 
         // Number of redundant UDP packets to send to increase test reliability
-        public const int UDPRedundancy = 10;
+        // Update: was 10, changing to 1 to measure impact of random test failures occurring on *nix.
+        // Certain random failures appear to be caused by a UDP client sending in a loop (based on UDPRedundancy)
+        // to a server which was closed but another server created (on a different thread \ test) that happens to
+        // have the same port #.
+        // This occurs on *nix but not Windows because *nix uses random values (1024-65535) while Windows increments.
+        public const int UDPRedundancy = 1;
```

Comment:
@ericeil added this in https://github.com/dotnet/corefx/pull/7112 to address https://github.com/dotnet/corefx/issues/5185.